lib/pi_piper/stub_driver.rb

```diff
@@ -89,6 +106,10 @@ def pin(pin_number)
         @pins[pin_number] || (@pins[pin_number] = {})
       end
 
+      def pwm(channel)
+        @pwm[channel] || (@pwm[channel] = {})
```

Comment:
My mistake, this should be removed.
It was an attempt to follow the specs of the Stubdriver.
But I quickly noticed that it doesn't test anything as it is now (we build a stub, psec it, but we don't spec the actual driver...)